Library/Homebrew/cmd/info.rb

```diff
@@ -375,7 +370,8 @@ def output_formula_analytics(f)
     full_analytics = args.analytics? || args.verbose?
 
     ohai "Analytics"
-    json["analytics"].each do |category, value|
+    json["analytics"].each do |category_, value|
+      category = category_.tr("_", "-")
```

Comment:
```suggestion
      category.tr!("_", "-")
```

Reply:
Tried that initially, but it throws `Error: can't modify frozen String`.